From a45603b0ff6ec04ebe4ab779d09ab146facfb77e Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:09 -0400
Subject: [redhat] kabi: Adding symbol seq_open (fs/seq_file.c)

Message-id: <e55009b29c07feb6a7caca9aa1418e12b42448c1.1410510773.git.poros@redhat.com>
Patchwork-id: 96137
O-Subject: [PATCH RHEL7.1 189/233] kabi: Adding symbol seq_open (fs/seq_file.c)
Bugzilla: 1071099
RH-Acked-by: Alexander Viro <aviro@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Initialize sequential file.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/seq_open
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/seq_open
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/seq_open

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/seq_open  b/redhat/kabi/kabi-module/kabi_ppc64/seq_open 
new file mode 100644
index 0000000..336c0f6
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/seq_open  
@@ -0,0 +1,2 @@
+#1-
+0x341529a3 seq_open vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/seq_open  b/redhat/kabi/kabi-module/kabi_s390x/seq_open 
new file mode 100644
index 0000000..cdf3f4f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/seq_open  
@@ -0,0 +1,2 @@
+#1-
+0x946cd9e3 seq_open vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/seq_open  b/redhat/kabi/kabi-module/kabi_x86_64/seq_open 
new file mode 100644
index 0000000..ab0227a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/seq_open  
@@ -0,0 +1,2 @@
+#1-
+0x257f3887 seq_open vmlinux EXPORT_SYMBOL
-- 
1.7.1